lang/eiffelstudio/Portfile

```
@@ -17,29 +17,23 @@ long_description  EiffelStudio is a development environment for the \
                   The user interface rests on a number of specific UI paradigms, \
                   in particular "pick-and-drop" for effective browsing.
 homepage          http://www.eiffel.com
```

Please make the `homepage` URL HTTPS (and any other links in the Portfile that can be).

```suggestion
homepage          https://www.eiffel.com
```